arch/arm64/boot/dts/qcom/sdm660-xiaomi-platina.dts

```diff
@@ -610,6 +627,16 @@
 	};
 };
 
+&pon {
+	resin {
+		compatible = "qcom,pm8941-resin";
+		interrupts = <0x0 0x8 1 IRQ_TYPE_EDGE_BOTH>;
+		debounce = <15625>;
+		bias-pull-up;
+		linux,code = <KEY_VOLUMEDOWN>;
+	};
+};
```

Comment:
instead of this `&pon` block you can just use already existing resin node label in pm660.dtsi ( https://github.com/sdm660-mainline/linux/blob/master/arch/arm64/boot/dts/qcom/pm660.dtsi#L67 ) and only override *some* properties, not all, like this for example:

```
&pon_resin  {
	status="okay";
	linux,code = <KEY_VOLUMEDOWN>;
}
```

something like that

Reply:
That seems reasonable. However, this resin node was added by 5.17 and doesn't exist in this branch. What should I do? 😵‍💫

Reply:
Right, I'm sorry, I linked master branch and didn't notice. You can

* ignore me here
* add resin node with label in this branch and use it

Reply:
I've done that.